frontend/src/metabase/lib/analytics.ts

```diff
@@ -0,0 +1,62 @@
+import * as Snowplow from "@snowplow/browser-tracker";
+
+import { shouldLogAnalytics } from "metabase/env";
+import Settings from "metabase/lib/settings";
+import type {
+  SchemaEvent,
+  SchemaType,
+  SimpleEvent,
+} from "metabase-types/analytics";
+
+export * from "./analytics-untyped";
+
+const VERSIONS: Record<SchemaType, string> = {
+  account: "1-0-2",
+  action: "1-0-0",
+  browse_data: "1-0-0",
+  cleanup: "1-0-0",
+  csvupload: "1-0-4",
```

Comment:
I will try to update it and push the changes

Reply:
Ok so looking into this once more it feels like it makes sense to entirely remove `1-0-4` and migrate the `csv_upload_clicked` event to the new generic event we added right away. 

Reasoning: The `1-0-4` makes a couple of fields optional - which is a breaking change in Redshift. We make it optional since we don't want to sent this information like csv file size etc.. Since it is just a click event, I think it is a perfect example for the generic events. 

So I think  this 

```
export const trackButtonClicked = () => {
  trackSchemaEvent("csvupload", {
    event: "csv_upload_clicked",
    source: "left_nav",
  });
};
```

Would need to change to 
```
export const trackButtonClicked = () => {
  trackEvent({
    event: "csv_upload_clicked",
    triggered_from: "left_nav",
  });
};
```


@ranquild would you mind giving this a try? I was about to change it but I am not sure whether I am overseeing something and we might also want to adjust those example types you added which we want to remove when adding the first generic event.

Reply:
@Somtom fixed